content/es/_includes/page-not-translated-msg.md

```diff
@@ -0,0 +1,7 @@
+---
+default_lang_commit: 1ececa0615b64c5dfd93fd6393f3e4052e0cc496
+---
+
+<i class="fa-solid fa-circle-info" style="margin-left: -1.5rem"></i> Estás
+viendo la **versión en inglés** de está página porque aún no ha sido traducida.
+¿Te interesa ayudar? Mira en [como contribuir](/docs/contributing/).
```

Comment:
nit: Maybe I would just say
```suggestion
¿Te interesa ayudar? Mira en [Contribuir](/docs/contributing/).
```

but the current wording sounds fine as well

Reply:
IMHO, `Mira en [como contribuir]` looks more natural.